src/cli.rs

```diff
@@ -25,6 +25,14 @@ pub struct Cli {
         help = "Delimiter for array values"
     )]
     pub delimiter: Option<Delimiter>,
+
+    #[clap(
+        global = true,
+        long,
+        value_name = "CR | LF | CRLF | Nul | Semicolon | String",
```

Comment:
One minor change: please remove the `Semicolon` argument as it does not exist.

Reply:
Ah will do, sorry I added this but actually ripped it out at the last moment since it seemed superfluous when general String is there.

- [ ] remove `Semicolon` in help text

Reply:
I thought so 😄